azure-pipelines.yml

```diff
@@ -0,0 +1,19 @@
+# Starter pipeline
```

Comment:
you'll need to remove this file and sign the CLA before we can review/merge the PR